SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc

```diff
@@ -33,56 +49,62 @@ ICS Statements per standard may be in a different subfolder and files - along wi
 
-#TODO: DOES ASCIIDOC NOT SUPPORT REPLACEMENT CHARACTERS (SEE NEXT TWO) OR IS IT DONE IN PUBLICATION TO HTML?#
+// Appendix B.x
+==== Overview of Standards Conformity Statements
+#TODO:  Provide a general overview of the topic, including some standards that provide Conformity statements, ICS tables, etc., and those that are moot on the topic - and how all of this is handled in this appendix and TF specification#
 
-* [[[ref_omg_sysml_2_0_spec,OMG SysML ^&#174;^ 2.0]]]  Systems Modeling Language &#8482; (SysML^&#174;^), Version 2.0, Object Management Group (OMG) specification.  Available at https://github.com/Systems-Modeling/SysML-v2-Release/tree/master/doc[github.com/Systems-Modeling/SysML-v2-Release/doc/]
+// ISO/IEEE 11073-10207:2017 ICS
 
-* [[[ref_omg_sysml_2_0_intro_graphical_model,OMG SysML ^&#174;^ Intro-Graphical Notation 2.0]]]  Systems Modeling Language&#8482;  (SysML^&#174;^), Version 2.0, Introduction to the Graphical Model, Object Management Group (OMG) standard.  Available at https://github.com/Systems-Modeling/SysML-v2-Release/tree/master/doc[github.com/Systems-Modeling/SysML-v2-Release/doc/]
+include::conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc[]
 
-* [[[ref_iec_60601_1_8_2020,IEC 60601-1-8:2020]]], IEC 60601-1-8:2006/AMD2:2020, Medical electrical equipment — Part 1-8: General requirements for basic safety and essential performance — Collateral standard: General requirements, tests and guidance for alarm systems in medical electrical equipment and medical electrical systems; Amendment 1:2020, and Amendment 2:2020. available at https://webstore.iec.ch/publication/59648[IEC Webstore] and the https://www.iso.org/standard/41986.html[ISO standards store].
+// Appendix B.x
+[bibliography]
+=== Bibliography
 
-* [[[ref_iec_80001_1_2021,ISO/IEC 80001-1:2021]]], IEC 80001-1:2021 Application of risk management for IT-networks incorporating medical devices — Part 1: Safety, effectiveness and security in the implementation and use of connected medical devices or connected health software.  Available at https://webstore.iec.ch/publication/34263[IEC Webstore] and https://www.iso.org/standard/72026.html[ISO standards store].
+In addition to Referenced Standards, which include normative requirements and capabilities that this technical framework utilizes, there are other standards, specifications, publications, presentations, materials, etc. that have proven valuable in both developing and understanding the framework's content.  This list identifies those items, including many of which are referenced throughout the specifications.
 
-* [[[ref_iso_81001_1_2021,ISO/IEC 81001-1:2021]]], ISO 81001-1:2021  Health software and health IT systems safety, effectiveness and security — Part 1: Principles and concepts.  Availabe at https://www.iso.org/standard/71538.html[ISO standards store] and https://webstore.iec.ch/publication/34286[IEC Webstore].
+==== Media (non-standards) References
```

Comment:
This block contains bib items and need to be marked accordingly in order to render bib refs correctly:
```suggestion
[bibliography]
==== Media (non-standards) References
```

Reply:
Inserted